### ISSUE TITLE
Question: How do I disable DASH Video from being used on Youtube

### ISSUE THREAD

**Issue by GregoriusT** [ytdl-org/youtube-dl#21360]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

WRITE QUESTION HERE

I have always gotten issues whenever I wanted to download a "fresh" Livestream from Youtube. And whenever I get that Issue I see that DASH Video is the Format being used.

Here an example of using -F:
139          m4a        audio only DASH audio   64k , m4a_dash container, mp4a.40.5@ 48k (24000Hz)
140          m4a        audio only DASH audio  144k , m4a_dash container, mp4a.40.2@128k (48000Hz)
160          mp4        256x144    DASH video  124k , mp4_dash container, avc1.42c00b, 30fps, video only
133          mp4        426x240    DASH video  258k , mp4_dash container, avc1.4d4015, 30fps, video only
134          mp4        640x360    DASH video  646k , mp4_dash container, avc1.4d401e, 30fps, video only
135          mp4        854x480    DASH video 1171k , mp4_dash container, avc1.4d401f, 30fps, video only
136          mp4        1280x720   DASH video 2326k , mp4_dash container, avc1.4d401f, 30fps, video only
298          mp4        1280x720   DASH video 3481k , mp4_dash container, avc1.4d4020, 60fps, video only
137          mp4        1920x1080  DASH video 4347k , mp4_dash container, avc1.640028, 30fps, video only
299          mp4        1920x1080  DASH video 5791k , mp4_dash container, avc1.64002a, 60fps, video only

Now Youtube does take about an hour to finish its transcoding and everything to get the Stream Footage working properly, and that moment is what I want to wait for whenever I am downloading new Videos from a Channel Page.

But the Issue I have is that the numbered Video Format IDs for DASH and for Normal are identical (which is also an issue), so I cant just blacklist those using -f. I also tried using [container!*=mp4_dash] but that does not do what I hoped it would do either.

Is there an Option/Parameter to prevent any "DASH video" and "DASH audio" type of thing from being accepted by a script as an available format, so that it will just say "format not available" instead of attempting to "desktop-lag, bandwidth-throttle and file-corrupt" the Video Download with Youtubes broken implementation of DASH?

I cant even watch those most Livestreams properly in latest Firefox, and people complain about Youtube having transcoding Issues, so I assume Youtube is at fault... but I still want my script to be robust against those types of Issues!

This is the -f Parameter I am using right now, which still causes the Issue.

"(243+249/243+250/243+139/243+171/243+140/243+worstaudio/134+249/134+250/134+139/134+171/134+140/134+worstaudio)/(bestvideo[height<=360]+worstaudio/best[height<=360]/18)[container!*=mp4_dash]"

I also use --match-filter '!is_live' in order to prevent Downloading of Livestreams, and the script is running in bash (Kubuntu).

I had lots of Issues with DASH Video in the past, just less severe, and I also updated youtube-dl just to see if that would have fixed my issue (which it doesn't). This is why I am asking for a way to just disable DASH Video/Audio from being used entirely, so my script can actually wait for Youtube to finish transcoding things, or even just not download the corresponding Videos ever, until I decide to try watching it on Youtube itself like a "normal person".

I would like to also avoid the "just download the videos 1 day later" solution, along with avoiding the "run youtube-dl twice, once for checking with -F if DASH is used which is a ton of complicated effort, and once for actually downloading"-Solution.

Note: I only say DASH Video/Audio because -F lists the Videos as such. Whenever -F does NOT list the Videos as such, it works just fine.

Tl;dr: What is the toggle/parameter/switch/option/command to avoid using what is labeled as DASH Video/Audio by -F? I cant find it in the ReadMe in any comprehensible way.

**Comment by GregoriusT** [comment]
Wait what? Not again, what is this a duplicate of?! I looked through the list of search results containing dash and there was nothing relevant in the 50 pages of Issue Titles I've seen!

I only asked for a switch or parameter that is not clearly documented in the Readme, or might not actually exist, and instead of getting a very short answer to a simple question, I get labeled commentless as a duplicate in an improper way again...

Thanks for telling me absolutely nothing, I guess.

**Comment by GregoriusT** [comment]
Okay Small Update, unsure if anyone is interested. Surely, people who search for this will be.

This doesn't work:
[container!^=dash]

This doesn't work either:
[protocol!=http_dash_segments]

But if you combine both of them, it will suddenly work and prevent dash video from being downloaded:
[protocol!=http_dash_segments][container!^=dash]

**Comment by Thupu** [comment]
@GregoriusT 

Hello,

I need to ask you about an issue with the Thorium Rods in FTB Ultimate 1.4.7 version of Minecraft. Before I got the new launcher from the FTB website, Thorium single rods would produce 1 EU/t. However now it does not produce constant energy, instead it delivers energy in one packet every 4-5 seconds, and the reading for energy output shows 0 EU/t for a while then 5 EU/t in a burst. This is not the way it was so something in the code between the times I used the old launchers files and now has changed. I really like what you have added to Minecraft and the fact you added Thorium is awesome.

I really need your help to fix this as my research showed your mod added Thorium to the game, and I don’t want this problem to prevent me from using it as it’s my favorite fuel source in the game, so much so I wrote a full 88 page 20,000+ word research paper on Nuclear Reactors in the game and even built Breeder reactors using Thorium as it is way easier to use then Uranium or Plutonium. 

If you could help I would greatly appreciate it, I have spent a lot of time trying to figure this out, then find a way to contact you, and this is my only chance I have of fixing this issue. 

Thank you for reading this,
Jared

**Comment by GregoriusT** [comment]
You are aware that this is so definitely not the place to ask something like this? Isn't there some way to message me directly? Heck even IRC would work and you wouldn't even need an account to ask something short like that...

And yes Thorium Rods years ago worked in bursts because otherwise there would have been big exploits.

Also don't reply to this Issue anymore just to talk with me, this is not the place to do that.

**Comment by Thupu** [comment]
Firstly you said on your contact page this was one way to contact you and I quote, 
https://gregtech.overminddl1.com/contact/
“GitHub
@GregoriusT mentioning this Username including the @ on any Issue will give me an E-Mail notification and I’ll likely reply to it.”

I take it from your reply that the Thorium Rods will no longer produce 1 EU/t and from here forward and will be Burst amounts of energy. And there’s really no need for such a condescending reply when I simply followed the appropriate procedures you offered. 

Reagrds.

**Comment by GregoriusT** [comment]
There you go, i changed it to also say ", unless you do so on a completely unrelated Issue."

You really need common sense instead of just searching a random post of me somewhere on GitHub and replying to it with completely unrelated content, you could have done something like message me directly.
